sdk/agentserver/tests/Azure.AI.AgentServer.Tests/Azure.AI.AgentServer.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RequiredTargetFrameworks>$(LtsTargetFramework)</RequiredTargetFrameworks>
+    <TargetFrameworks>$(LtsTargetFramework)</TargetFrameworks>
+
+    <!-- We don't care about XML doc comments on test types and members -->
+    <NoWarn>$(NoWarn);CS1591;SA1010;SA1200;SA1600;</NoWarn>
+  </PropertyGroup>
+
+  <!-- Reference the Client Library -->
+  <ItemGroup>
+    <ProjectReference Include="$(AzureCoreTestFramework)"/>
+
+    <ProjectReference Include="..\Agents.Customized.Integration.Tests\Agents.Customized.Integration.Tests.csproj"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="coverlet.collector" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="NUnit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="8.*" />
```

Comment:
You should not be using version overrides without a special case.   If we need to bump a central version for some of these ASPNET packages, let's look at the viability of that that first.